src/workloads/query/TimeseriesStressUnpacking.yml

```diff
@@ -0,0 +1,228 @@
+              OperationCommand:
+                # With 600 documents per bucket, 10 buckets will have 6000 <= 'measurement0' <= 12000 (6000 + 6000).
+                Pipeline:
+                  [
+                    {
+                      $match:
+                        {
+                          $and:
+                            [
+                              { "measurement0": { $gt: 6000 } },
+                              { "measurement0": { $lte: 12100 } },
+                            ],
+                        },
+                    },
+                  ]
+
+  - Name: MatchTarget100Buckets
+    Type: CrudActor
+    Database: *database
+    Threads: 10
+    Phases:
+      OnlyActiveInPhases:
+        Active: [6]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationMetricsName:
+              OperationName: aggregate
+              OperationCommand:
+                # With 600 documents per bucket, 100 buckets will have 'measurement0' <= 60000 (600 * 100).
+                Pipeline: [{ $match: { "measurement0": { $gt: 60100 } } }]
+
+  - Name: ConjunctionMatchMeasurement100Buckets
+    Type: CrudActor
+    Database: *database
+    Threads: 10
+    Phases:
+      OnlyActiveInPhases:
+        Active: [7]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationMetricsName:
+              OperationName: aggregate
+              OperationCommand:
+                # With 600 documents per bucket, 100 buckets will have 6000 <= 'measurement0' <= 66000 (6000 + 600 * 100).
+                Pipeline:
+                  [
+                    {
+                      $match:
+                        {
+                          $and:
+                            [
+                              { "measurement0": { $gt: 6000 } },
+                              { "measurement0": { $lte: 66100 } },
+                            ],
+                        },
+                    },
+                  ]
+
+  - Name: MatchTarget1000Buckets
+    Type: CrudActor
+    Database: *database
+    Threads: 10
+    Phases:
+      OnlyActiveInPhases:
+        Active: [8]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationMetricsName:
+              OperationName: aggregate
+              OperationCommand:
+                # With 600 documents per bucket, 1000 buckets will have 'measurement0' <= 600000 (600 * 1000).
+                Pipeline: [{ $match: { "measurement0": { $gte: 600100 } } }]
+
+AutoRun:
+  - When:
+      mongodb_setup:
+        $eq:
+          - standalone-all-feature-flags
```

Comment:
This variant is picked so we target the current SBE project. I think when the project is done we should update this to include replica set. Does that sound reasonable? The same applies for the variant in the other workload.